packages/common/client-utils/package.json

```diff
@@ -22,13 +22,13 @@
 		"build": "fluid-build . --task build",
 		"build:commonjs": "fluid-build . --task commonjs",
 		"build:compile": "fluid-build . --task compile",
-		"build:docs": "api-extractor run --local --typescript-compiler-folder ./node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
```

Comment:
(Note applicable to all packages): Copying is now done by `build:gendocs` in the root package.json, so that logic could be removed. Additionally, we explicitly force api-extractor to use **our** version of typescript via a pnpm override, so we don't need these path overrides either.